### PR TITLE
Proper interval rounding on normal trends

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- the first interval on weekly or monthly interval wouldn't always be accurate because the date comparison wasn't properly rounded when applied

*If this affects the frontend, include screenshots.*  

## Checklist

- [ ] need to make similar adjustment to breakdown query
- [ ] cover with tests

### CODE REVIEW

File: ee/clickhouse/queries/util.py [hunk 1]
```diff
@@ -41,7 +42,7 @@ def format_ch_timestamp(timestamp: datetime, filter, default_hour_min: str = " 0
         or (filter._date_from == "-24h")
         or (filter._date_from == "-48h")
     )
-    return timestamp.strftime("%Y-%m-%d{}".format(" %H:%M:%S.%f" if is_hour_or_min else default_hour_min))
+    return timestamp.strftime("%Y-%m-%d{}".format(" %H:%M:%S" if is_hour_or_min else default_hour_min))
```
Comment (EDsCODE) [inline ee/clickhouse/queries/util.py:45]: @timgl was there a reason we needed microseconds here?

Comment (timgl) [reply ee/clickhouse/queries/util.py:45]: Yes! we use this to do ?after= on events list. If it's only precise to seconds we might miss some events in that list.

Comment (EDsCODE) [reply ee/clickhouse/queries/util.py:45]: ah i see let me work that back in

Comment (EDsCODE) [reply ee/clickhouse/queries/util.py:45]: oh the event conditions for event list are separate so this doesn't affect it which is why no tests failed

Comment (timgl) [reply ee/clickhouse/queries/util.py:45]: Ah great, in that case go ahead

**Pull request merged**